recipes/dustmaps/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dustmaps" %}
+{% set version = "1.0" %}
+{% set sha256 = "f9099f12b84aa28959b171ea9057f178511595c50db9e3b1775e1fb924910108" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
The preferred magic install line is now:

```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```


([Cf docs](https://conda-forge.org/docs/meta.html#use-pip))